src/WebrootPreloadMiddleware.php

```diff
@@ -94,12 +94,13 @@ public function __invoke(ServerRequestInterface $request, callable $next)
             }
 
             if ($request->hasHeader('If-Match')) {
-                $expectedEtag = current($request->getHeader('If-Match'));
-                $expectedEtag = trim($expectedEtag, '"');
-                if ($expectedEtag !== $item['etag']) {
-                    return new Response(412);
+                foreach($requested->gethHeader('if-match') as $expectedEtag) {
+                        $expectedEtag = trim($expectedEtag, '"');
+                   if($expectedEtag !== $item['etag']){
+                     return new Response(412);
+                  }
```

Comment:
Could you fix the intending here, the easiest way would be to run `make cs-fix` in your bash/zsh shell, alternatively PrettyCI lists the changes here: https://github.com/WyriHaximus/reactphp-http-middleware-webroot-preload/pull/16/checks?check_run_id=27180248